### PR TITLE
Add QCOM LT distro project/testplan

### PR DESCRIPTION
**Pull request by alimon**

### CODE REVIEW

**Comment by alimon** [comment]
@roxell Can you take a look?, Thanks.

File: testcases/distro-smoke.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+{% set test_name = test_name | default("distro-smoke") %}
+
+{% block test_target %}
+  {{ super() }}
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/smoke/smoke.yaml
+      params:
+              TESTS: {{SMOKE_TESTS}}
```
Comment (roxell) [inline testcases/distro-smoke.yaml:16]: remove some spaces

File: testcases/distro-smoke.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/dd-wr-speed/dd-wr-speed.yaml
+      name: linux-dd-wr-speed
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/sysbench/sysbench.yaml
+      params:
+              NUM_THREADS: 4
+              TESTS: cpu memory threads mutex
```
Comment (roxell) [inline testcases/distro-smoke.yaml:44]: remove some spaces

File: testcases/distro-smoke.yaml (hunk 1 above)
Comment (alimon) [reply testcases/distro-smoke.yaml:16]: @roxell Done.

File: testcases/distro-smoke.yaml (hunk 2 above)
Comment (alimon) [reply testcases/distro-smoke.yaml:44]: @roxell Done.

File: testcases/distro-smoke.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+      name: linux-toolchain-smoke
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/device-tree/device-tree.yaml
+      name: linux-device-tree
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/meminfo/meminfo.yaml
+      name: meminfo
+
```
Comment (roxell) [inline testcases/distro-smoke.yaml:34]: what's the reason for the newlines in random places in this file?

**Comment by roxell** [comment]
When will you drop lxc and move to docker?
LXC is deprecated in LAVA, so I would like to see use move away from LXC containers.

File: testcases/distro-smoke.yaml (hunk 3 above)
Comment (alimon) [reply testcases/distro-smoke.yaml:34]: Any reason, at all, if you like i Can remove.

**Comment by alimon** [comment]
> When will you drop lxc and move to docker? LXC is deprecated in LAVA, so I would like to see use move away from LXC containers.

I know, Maria told me about that idea, Any info about the schedule to drop LXC?.

File: testcases/distro-smoke.yaml (hunk 3 above)
Comment (roxell) [reply testcases/distro-smoke.yaml:34]: yeah please drop it.

**Comment by roxell** [comment]
they have been asking us to drop it for over a year now... I think they are waiting until we drop it before they drop LXC, since they don't want to break things.
Thats why I think its something we have to move away from ASAP.

File: testcases/distro-smoke.yaml (hunk 3 above)
Comment (alimon) [reply testcases/distro-smoke.yaml]: @roxell Done.

**Comment by alimon** [comment]
> they have been asking us to drop it for over a year now... I think they are waiting until we drop it before they drop LXC, since they don't want to break things. Thats why I think its something we have to move away from ASAP.

yes i have some schedule for this quarter, i will try to make the migration before end of this year.

**Comment by roxell** [comment]
> > they have been asking us to drop it for over a year now... I think they are waiting until we drop it before they drop LXC, since they don't want to break things. Thats why I think its something we have to move away from ASAP.
> 
> yes i have a schedule for this quarter, i will try to make the migration before end of this year.

That sounds great.

**Pull request merged**